acquia/acquia_cms
# ACMS-706: Fix tag links in firefox.

**Motivation**
- This PR fix the clicking issue in firefox for tags on listing pages.

**Proposed changes**
 - End  user will be able to click on Tags on event listing pages in all the browsers including  Firefox.
 - Increased z-index of tag link to work in Firefox. 

**Alternatives considered**


**Testing steps**
- Enable Starter module if not enabled. 
- Open Firefox  browser 
- Go to /articles page
- Verify you are able to click on Tags.
- Verify the  functionality for other pages also like. /events, /places, /people

**Merge requirements**
